### DISCUSSION TITLE
New Source: Pennylane

### DISCUSSION THREAD

**Discussion by benlienart** [airbytehq/airbyte#33174] [New Connector Request]
## Tell us about the new connector you’d like to have
Source connector to retrieve data from [Pennylane API](https://pennylane.readme.io/reference/presentation) and the following streams : 
* [customers](https://pennylane.readme.io/reference/customers-post-1)
* [suppliers](https://pennylane.readme.io/reference/suppliers-get)
* [customers invoices](https://pennylane.readme.io/reference/customer_invoices-import-1)
* [suppliers invoices](https://pennylane.readme.io/reference/supplier_invoices-get)

## Describe the context around this new connector
The connector will be used to retrieve data linked to a [Pennylane](https://www.pennylane.com/) account, for P&L reporting purposes : "Pennylane is the key financial management platform for SMEs in Europe."

Connection could be done using full-refresh or incremental sync, as there is date fields that can be used for this purpose.

My company Garlic is a Pennylane customer and we can have access to their sandbox and production API environments.

## Are you willing to submit a PR?
I am already working on a Airbyte CDK-based  connector and will be happy to submit a PR. @YowanR can you assign the PR to me ?

**Comment by Cnstant** [comment]
Hello, 
I could use such a connector, and I'd gladly help to develop it. Is it still on ?

[a comment by an automated account is not shown]
